llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -198,13 +216,21 @@ bool RISCVInstructionSelector::select(MachineInstr &MI) {
 
   switch (Opc) {
   case TargetOpcode::G_ANYEXT:
-  case TargetOpcode::G_TRUNC:
+  case TargetOpcode::G_PTRTOINT:
+  case TargetOpcode::G_TRUNC: {
     MI.setDesc(TII.get(TargetOpcode::COPY));
+    bool selected = selectCopy(MI, MRI);
```

Comment:
`selected` -> `Selected` ("Variable names should be nouns (as they represent state). The name should be camel case, and start with an upper case letter (e.g. Leader or Boats).")